portugues/toalha/carona/carona.md

```diff
@@ -0,0 +1 @@
+Você consegue a carona e descobre que a coisa amarela é uma nave dos Vogons, uma raça alienígena um tanto bisonha. Infelizmente, os vogons são conhecidos como os quarto piores programadores da galáxia, e não tem a menor chance de resolver o bug. Acho que você vai ter que escutar poesia por alguns anos ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.